### PR TITLE
updated the environments with python version for sktime, added python 3.9

### PR DESCRIPTION
**Pull request by Vasudeva-bit**
#### Reference Issues/PRs

Not related to any fixe/issue, just updating the environments for python versions (sktime), added 3.9

#### What does this implement/fix? Explain your changes.

just updating compatible python versions for sktime
#### Does your contribution introduce a new dependency? If yes, which one?
No dependency is introduced

#### What should a reviewer concentrate their feedback on?
cross-check that sktime works with python 3.9

##### For all contributions
- [ ] I've added myself to the [list of contributors](https://github.com/alan-turing-institute/sktime/blob/main/.all-contributorsrc).

### CODE REVIEW

File: docs/source/get_started.rst [hunk 1]
```diff
@@ -11,7 +11,7 @@ Installation
 
 ``sktime`` currently supports:
 
-* environments with python version 3.6, 3.7, or 3.8.
+* environments with python version 3.6, 3.7, 3.8, or 3.9.
```
Comment (fkiraly) [inline docs/source/get_started.rst:14]: thanks!
3.6 is no longer supported, so that should actually be removed.
Thanks for spotting!

**Review by fkiraly — CHANGES_REQUESTED** [review]
Thanks for this, this was indeed a typo!

The "hierarchical" function has already been renamed, so once you update from `main`, it shoudl be in your PR.

Re support, 3.6 is no longer supported, that should be removed, too.

File: docs/source/get_started.rst (hunk 1 above)
Comment (Vasudeva-bit) [reply docs/source/get_started.rst:14]: oh okay, I will do this small fix!

**Review by fkiraly — APPROVED** [review]
Thanks for updating this!

**Pull request merged**